test/conformance/chainsaw/generate/clusterpolicy/standard/existing/resourcequota-recreation-with-preconditions/policy.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: resourcequota-recreation-with-preconditions
+spec:
+  validationFailureAction: Enforce
+  background: true
+  rules:
+  - name: create-default-quota
+    match:
+      any:
+      - resources:
+          kinds:
+          - Namespace
+    context:
+      - name: resourcequotas
+        apiCall:
+          urlPath: "/api/v1/namespaces/{{request.object.metadata.name}}/resourcequotas"
+          jmesPath: "items[?metadata.name==`override`] | length(@)"
```

Comment:
```suggestion
          jmesPath: "items[?metadata.name=='override'] | length(@)"
```

Reply:
It reported a syntax [error](https://github.com/kyverno/kyverno/actions/runs/18187779631/job/51776767358?pr=13965#step:3:901):

```
        admission webhook "validate-policy.kyverno.svc" denied the request: path: spec.rules[0]: failed to parse JMESPath items[?metadata.name==`override`] | length(@): invalid character 'o' looking for beginning of value
```